src/pages/studyView/StudyViewPage.tsx

```diff
@@ -1,27 +1,23 @@
 import * as React from 'react';
 import {inject, observer} from "mobx-react";
-import styles from "./styles.module.scss";
-import { MSKTab, MSKTabs } from "../../shared/components/MSKTabs/MSKTabs";
-import { reaction, observable, computed } from 'mobx';
-import { StudyViewPageStore } from 'pages/studyView/StudyViewPageStore';
-import {CancerStudy} from 'shared/api/generated/CBioPortalAPI';
+import {MSKTab, MSKTabs} from "../../shared/components/MSKTabs/MSKTabs";
+import {computed, reaction} from 'mobx';
+import {StudyViewPageStore} from 'pages/studyView/StudyViewPageStore';
 import LoadingIndicator from "shared/components/loadingIndicator/LoadingIndicator";
 import {ClinicalDataTab} from "./tabs/ClinicalDataTab";
-import setWindowVariable from "../../shared/lib/setWindowVariable";
 import * as _ from 'lodash';
 import ErrorBox from 'shared/components/errorBox/ErrorBox';
 import getBrowserWindow from "../../shared/lib/getBrowserWindow";
 import 'react-grid-layout/css/styles.css';
 import 'react-resizable/css/styles.css';
-import {stringListToSet} from "../../shared/lib/StringUtils";
-import classnames from 'classnames';
-import {buildCBioPortalPageUrl} from 'shared/api/urls';
-import MobxPromise from 'mobxpromise';
-import { StudySummaryRecord } from 'pages/studyView/virtualStudy/VirtualStudy';
 import {PageLayout} from "../../shared/components/PageLayout/PageLayout";
 import IFrameLoader from "../../shared/components/iframeLoader/IFrameLoader";
-import { StudySummaryTab } from 'pages/studyView/tabs/SummaryTab';
-import {StudyViewFilter} from "../../shared/api/generated/CBioPortalAPIInternal";
+import {StudySummaryTab} from 'pages/studyView/tabs/SummaryTab';
+import StudyPageHeader from "./studyPageHeader/StudyPageHeader";
+import "./styles.scss";
+import styles from './styles.module.scss';
+import SelectedInfo from "./SelectedInfo/SelectedInfo";
```

Comment:
may be combining both the style files

Reply:
We cannot yet. There is a global style class, this is the only way to overwrite some styles.